src/ucp/core/ucp_context.h

```diff
@@ -131,6 +131,8 @@ typedef struct ucp_context_config {
     size_t                                 estimated_num_ppn;
     /** Enable flushing endpoints while flushing a worker */
     int                                    flush_worker_eps;
+    /** Enable shuffling endpoints while creating a worker */
```

Comment:
w/o `enable`, this one doesn't "enable".

Reply:
done